rest_registration/locale/fr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,108 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
I think we should update the metadata here - this looks like auto-generated stuff; you could put yourself as the author of the translation.